transactions/views.py

```diff
@@ -368,14 +370,14 @@ def validate_payment(request):
 
 
 @api_view(['GET'])
-@permission_classes((IsAuthenticated, ))
 def foreign_card_validation_response(request):
     """
     Endpoint for handling foreign card validation response. It uses the
     response to verify the payment and save the card token, if requested
     :param request: DRF request object
     :return: JSON response
     """
+    domain = os.environ.get('FRONT_END_LOGIN_URL')
```

Comment:
Hey @patrickf949, Just to understand here, Are we redirecting to the login URL or some other part of our frontend that is protected?

Reply:
@CryceTruly we are redirecting to the frontend Url.
The variable name will be changed